l1-contracts/contracts/bridge/L1SharedBridge.sol

```diff
@@ -768,7 +774,8 @@ contract L1SharedBridge is IL1SharedBridge, ReentrancyGuard, Ownable2StepUpgrade
             _l2BatchNumber: _l2BatchNumber,
             _l2MessageIndex: _l2MessageIndex,
             _l2TxNumberInBatch: _l2TxNumberInBatch,
-            _merkleProof: _merkleProof
+            _merkleProof: _merkleProof,
+            _isLegacyEncoding: true
```

Comment:
does it mean that the current SDK will stop working for new deposits?

Reply:
updated so it would work